docs/en/plugins/php_loc.md

```diff
@@ -6,12 +6,12 @@ Runs [PHPLoc](https://github.com/sebastianbergmann/phploc) against your project
 Configuration
 -------------
 
+See also [Common Plugin Configuration Options](../plugin_common_options.md).
+
 ### Options
 
-* **directory** - Optional - The directory in which phploc should run. 
-* **binary_name** [string|array, optional] - Allows you to provide a name of the binary
-* **binary_path** [string, optional] - Allows you to provide a path to the binary
-* **ignore** 
+None, except [Common Plugin Configuration Options](../plugin_common_options.md)
```

Comment:
I don't like these _See also..._ and _None, except..._ with same link here and for few other plugins.

I would leave one of them.